features/admin.users.v1/models/user.ts

```diff
@@ -343,7 +343,10 @@ export interface PayloadInterface {
             display: string;
             value: string;
           }[];
-        };
+        } | {
```

Comment:
this is v3 api payload format I guess. If yes, shall we define a separate interface for that for better clarity?

Reply:
This has been fixed: https://github.com/wso2/identity-apps/pull/8184/commits/364f88e7a85aa677d870962f791af4ab219be515